src/css/components/icon.scss

```diff
@@ -24,28 +24,34 @@
 .icon {
   display: inline-block;
   fill: transparent;
-  position: relative;
+  height: 1.5em;
+  max-height: $grid-4;
+  max-width: $grid-4;
+  min-height: $grid-2;
```

Comment:
Do we need to set a max height, I'm wondering what that would do when the panel is larger?